web/src/Root.tsx

```diff
@@ -87,9 +83,7 @@ type State = {
   appNameSpace: string | null;
   appsList: App[];
   appSlugFromMetadata: string | null;
-  adminConsoleMetadata: {
-    isKurl?: boolean;
-  };
+  adminConsoleMetadata: Metadata | null;
```

Comment:
Metadata type has been added as a global type. it's the type for `/metadata` response